src/index.ts

```diff
@@ -1816,8 +1819,10 @@ async function displayActivePool(poolParams: PoolParams|PoolParamsP3|PoolParamsN
       displayIfTokenPool(newPool, isAccountRegistered)
 
     } else if(poolParams instanceof PoolParamsNFT) {
-
-      displayIfNftPool(newPool, isAccountRegistered)
+      const poolUserStatus = await poolParams.stakingContractData.getUserStatus()
+      // check for user stake 
+      const hasUserStakedNFT = poolUserStatus.stake_tokens.find(total => total?.length > 0)?.length && poolUserStatus.stake != "0" ? true : false

```

Comment:
`poolUserStatus.stake_tokens.find(total => total?.length > 0)?.length && poolUserStatus.stake != "0"` this is already a boolean. There is no need for the `? true : false`. It would be the same code. 

Also, the ? in total shouldn't be needed. `stake_tokens` is an array of arrays (please, change that in the object), so total will always be an array.

Reply:
okay will update the code 
also I checked in `PoolUserStatusP3NFT` its `stake_tokens: string[][] = [];` only